components/AboutUs/index.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { useState } from 'react';
+import React, { useState, useRef } from 'react';
 import Image from 'next/image';
 import { TEXT_CONSTANTS } from 'types/enum';
 
 const AboutUs = () => {
   const [progress] = useState(60);
+  const aboutUs = useRef<HTMLInputElement>(null);
   return (
-    <div className=" relative  w-full h-screen  ">
+    <div className=" relative  w-full h-1/2" ref={aboutUs} id="aboutUs">
```

Comment:
Why ref and id?

Reply:
> Why ref and id?

for moving a perticular section within same page... like aboutus section

Reply:
> Why ref and id?

for moving a perticular section within same page... like aboutus section